src/clientSideScene/sceneEntities.ts

```diff
@@ -141,12 +141,14 @@ export class SceneEntities {
   activeSegments: { [key: string]: Group } = {}
   intersectionPlane: Mesh | null = null
   axisGroup: Group | null = null
+  draftPointGroups: Group[] = []
   currentSketchQuaternion: Quaternion | null = null
   constructor(engineCommandManager: EngineCommandManager) {
     this.engineCommandManager = engineCommandManager
     this.scene = sceneInfra?.scene
     sceneInfra?.camControls.subscribeToCamChange(this.onCamChange)
     window.addEventListener('resize', this.onWindowResize)
+    this.createIntersectionPlane()
```

Comment:
Nice - am I close in understanding that instead of re-creating the raycasting plane many times, we create it once and let it live for the application lifetime?

Reply:
Correct. We had logic that determined the lifecycle of this intersection plane within the three.js scene we created.
We do not need to add and remove it because it is safe to leave in the scene the entire time. 

One of the bugs was the mouse events worked but couldn't raycast the plane because it wasn't in the scene because the logic to add and remove it was broken. 

Instead of fixing that workflow logic which is in several disjoint workflows I leave it in the scene the entire time because it is safe.